src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -565,6 +565,24 @@ public void TestNormalizeTuples()
             TestNormalizeDeclaration("public (string prefix,string uri)Foo()", "public (string prefix, string uri) Foo()");
         }
 
+        [Fact]
+        [WorkItem(46656, "https://github.com/dotnet/roslyn/issues/46656")]
+        public void TestNormalizeBlockLambdas()
+        {
+            TestNormalizeStatement("_=()=>{};", "_ = () =>\r\n{\r\n};");
+            TestNormalizeStatement("_=x=>{};", "_ = x =>\r\n{\r\n};");
+            TestNormalizeStatement("Add(()=>{});", "Add(() =>\r\n{\r\n});");
+            TestNormalizeStatement("Add(()=>{{_=x=>{};}});",
+@"Add(() =>
+{
+  {
+    _ = x =>
+    {
+    };
+  }
+});");
+        }
```

Comment:
Would prefer a theory here. That way if one fails it's trivial to just run that config